server/middleware/tchannel-client/helpers/make-request.js

```diff
@@ -25,16 +25,18 @@ const formatMethod = require('./format-method');
 const formatRequestName = require('./format-request-name');
 const uiTransform = require('./ui-transform');
 
-const makeRequest = ({ authTokenHeaders, channel, ctx }) => ({
+const makeRequest = ({ authTokenHeaders, channels, ctx }) => ({
```

Comment:
do we use the same auth token for the both channels?

Reply:
It's a good question. I need to be able to test this in staging to know whether we need different credentials. Right now I'm not sure.

Reply:
It shouldn't be a massive change in the event it does need separate credentials.